src/tests/class_tests/openms/source/MRMTransitionGroupPicker_test.cpp

```diff
@@ -358,41 +491,41 @@ START_SECTION((template <typename SpectrumT, typename TransitionT> MRMFeature cr
     TEST_REAL_SIMILAR(mrmfeature.getFeature("2").getIntensity()/mrmfeature.getFeature("1").getIntensity(), 8.44773) // ratio should be stable
   }
 
-  // background subtraction with peak integrator
+  // integration with Simpon's rule (with background subtraction)
   {
     MRMTransitionGroupPicker picker;
     Param picker_param = picker.getDefaults();
     picker_param.setValue("PeakPickerMRM::method", "legacy"); // old parameters
     picker_param.setValue("PeakPickerMRM::peak_width", 40.0); // old parameters
     picker_param.setValue("background_subtraction", "exact");
+    picker_param.setValue("PeakIntegrator:integration_type", "simpson");
     picker.setParameters(picker_param);
 
     MRMFeature mrmfeature = picker.createMRMFeature(transition_group, picked_chroms, smoothed_chroms, chr_idx, peak_idx);
 
-    TEST_REAL_SIMILAR(mrmfeature.getFeature("2").getIntensity(), 514590); // slightly reduced value due to background subtraction
+    TEST_REAL_SIMILAR(mrmfeature.getFeature("2").getIntensity(), 1.42194e+06); // slightly reduced value due to background subtraction
     TEST_REAL_SIMILAR((double)mrmfeature.getFeature("2").getMetaValue("peak_apex_int"), 155331.37806798); // slightly reduced value due to background subtraction
-    TEST_REAL_SIMILAR(mrmfeature.getFeature("1").getIntensity(), 61009.7); // slightly reduced value due to background subtraction
+    TEST_REAL_SIMILAR(mrmfeature.getFeature("1").getIntensity(), 168685); // slightly reduced value due to background subtraction
     TEST_REAL_SIMILAR((double)mrmfeature.getFeature("1").getMetaValue("peak_apex_int"), 30251.2414481247); // slightly reduced value due to background subtraction
-    TEST_REAL_SIMILAR(mrmfeature.getFeature("2").getIntensity()/mrmfeature.getFeature("1").getIntensity(), 8.43456040596823) // ratio should be stable
+    TEST_REAL_SIMILAR(mrmfeature.getFeature("2").getIntensity()/mrmfeature.getFeature("1").getIntensity(), 8.42957) // ratio should be stable
   }
 
-  // integration with Simpon's rule (with background subtraction)
+  // background subtraction with peak integrator
   {
     MRMTransitionGroupPicker picker;
     Param picker_param = picker.getDefaults();
     picker_param.setValue("PeakPickerMRM::method", "legacy"); // old parameters
     picker_param.setValue("PeakPickerMRM::peak_width", 40.0); // old parameters
     picker_param.setValue("background_subtraction", "exact");
-    picker_param.setValue("PeakIntegrator:integration_type", "simpson");
     picker.setParameters(picker_param);
 
     MRMFeature mrmfeature = picker.createMRMFeature(transition_group, picked_chroms, smoothed_chroms, chr_idx, peak_idx);
 
-    TEST_REAL_SIMILAR(mrmfeature.getFeature("2").getIntensity(), 1.42194e+06); // slightly reduced value due to background subtraction
```

Comment:
also a large change hier

Reply:
That was changed in a previous PR for this class.  It is showing up in the diff because I had done a copy and paste operation on those lines by accident.  I did not change the code that was there previously.

Reply:
see line 501, I think @dmccloskey just switched the two tests around so that the simpson integration is now first.